module1.js

```diff
@@ -65,6 +80,7 @@ function question4(array) {
 
 function question5(number1, number2) {
   // TODO:
+  return number1 * number2;
```

Comment:
Yep! One line of code - this is your clue for fixing questions 1 and 2 which can be both written on one line as well :)